docs/content-services/components/node-comments.component.md

```diff
@@ -27,3 +28,9 @@ Displays comments from users involved in a specified content and allows an invol
 | ---- | ---- | ------------- | ----------- |
 | nodeId | `string` |  | nodeId of the document that has comments |
 | readOnly | `boolean` |  | make the [comments component](../../core/components/comments.component.md) readOnly |
+
+### Events
+
+| Name         | Type                                                            | Description                                        |
+|--------------|------------------------------------------------------------------|----------------------------------------------------|
+| commentAdded | [`EventEmitter`](https://angular.io/api/core/EventEmitter)<CommentModel> | Emitted when a new comment is successfully added. |
```

Comment:
Also here - could you please format it?
<img width="1233" alt="image" src="https://github.com/user-attachments/assets/52138f14-676e-4e6e-8e0f-d5aec4385276" />

It should to look like: 
<img width="1424" alt="image" src="https://github.com/user-attachments/assets/10a91180-80b7-4e04-b942-89e77b3519e4" />